src/foundry/foundry.js/pixi/polygons/pointSourcePolygon.d.ts

```diff
@@ -0,0 +1,97 @@
+interface PointSourcePolygonConfig {
+  /** The type of polygon being computed */
+  type: foundry.CONST.WALL_RESTRICTION_TYPES;
+
+  /**
+   * The angle of emission, if limited
+   * @defaultValue `360`
+   */
+  angle: number;
```

Comment:
according to foundry's JSDoc, this and most of the other properties are optional. Why aren't they optional here?